incubator/zalenium/README.md

```diff
@@ -0,0 +1,125 @@
+# ZALENIUM-HELM-CHART
+======================================================
+
+## Introduction
+
+This chart bootstraps a zalenium deployment on a kubernetes cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- teracy-dev v0.6.0-a4+ (https://github.com/teracyhq/dev)
+- Follow this doc (https://github.com/teracyhq-incubator/teracy-dev-entry-k8s) to set up teracy-dev.
```

Comment:
"Follow the docs on https://github.com/teracyhq-incubator/teracy-dev-entry-k8s to set up a local k8s cluster."

The docs on teracy-dev-entry-k8s will help users to set up a working local k8s cluster, if the docs is not good enough, improve it on the right project.